scripts/airports.ts

```diff
@@ -0,0 +1,165 @@
+import fs from "fs";
+import _ from "lodash";
+import countriesTranslation from "i18n-iso-countries";
+import { z } from "zod";
+
+const germanLocale = "de";
```

Comment:
Since the name of the other constants is capitalized, shall we also capitalize the name of this const:

```suggestion
const GERMAN_LOCALE = "de";
```

?